games-action/ddnet/ddnet-2023.03.09.recipe

```diff
@@ -0,0 +1,127 @@
+SUMMARY="A free cooperative platformer game based on Teeworlds"
+DESCRIPTION="DDraceNetwork (DDNet) is an actively maintained version of \
+DDRace, a Teeworlds modification with a unique cooperative gameplay. Help \
+each other play through custom maps with up to 64 players, compete against \
+the best in international tournaments, design your own maps, or run your own \
+server. The official servers are around the world. All ranks made on official \
+servers are available worldwide and you can collect points!"
+HOMEPAGE="https://ddnet.org"
+COPYRIGHT="2007-2014 Magnus Auvinen \
+	2010-2011 Shereef Marzouk \
+	2013-2022 Dennis Felsing"
+LICENSE="Zlib"
```

Comment:
```suggestion
LICENSE="Zlib
	CC-BY-SA-3.0
	SIL Open Font License v1.1"
```

Please add a copy of the license CC-BY-SA-3.0 like with https://github.com/haikuports/haikuports/blob/2490464cd13f580060acebc18ef617f55e30da44/games-action/lugaru/licenses/CC-BY-SA-3.0